### DISCUSSION TITLE
第1节课笔记(22班)

### DISCUSSION THREAD

**Discussion by BarryYin** [InternLM/Tutorial#156] [笔记]
请大家添加笔记地址URL

**Comment by jiayaozhang** [comment]
https://github.com/jiayaozhang/InternLM_Notes

**Comment by Juanting-Xu** [comment]
https://github.com/Juanting-Xu/InternLM_notes

**Reply by Juanting-Xu** [reply]
更新：https://github.com/Juanting-Xu/InternLM_notes/blob/main/lecture1.md

**Comment by hyanworkspace** [comment]
https://github.com/hyanworkspace/InternLM/blob/main/notes/lecture1.md

**Comment by cavedweller509** [comment]
https://github.com/BestAnHongjun/InternLM_Notes

**Reply by cavedweller509** [reply]
更新：https://github.com/BestAnHongjun/InternLM_Notes/blob/main/notes/lecture_1.md

**Comment by TAICHIKF** [comment]
https://github.com/TAICHIKF/InternLM_notes.git

**Comment by Jasimyn** [comment]
https://broadleaf-gemini-274.notion.site/Lecture1-98459b76713546cb8adb279a3047b815

**Comment by lilxmx** [comment]
https://blog.csdn.net/L1142456160/article/details/135491993

**Comment by Lynn401** [comment]
https://daisy-401.notion.site/b8f9767ba1334268b4a53664c7f25437?pvs=4

**Comment by CDL0726** [comment]
https://github.com/CDL0726/InternLM_Notes

**Comment by ZTianle** [comment]
https://www.notion.so/Lecture-1-ce36994cc0b34dcbbd50de46ca25d4c4?pvs=4

**Comment by goldwater2020** [comment]
https://github.com/goldwater2020/InternLM/blob/main/note/day1.md

**Comment by miyc1996** [comment]
https://github.com/miyc1996/InternLM/blob/main/lesson1.ipynb

**Comment by MinghanJia** [comment]
https://github.com/MinghanJia/InternLM/blob/main/Lecture1.md

**Comment by aqizhoua** [comment]
https://aqizhoua.github.io/poster/a8a37251.html

**Comment by Osmar99695** [comment]
https://github.com/Osmar99695/InternLM/blob/main/README.md

**Comment by lankuohsing** [comment]
https://github.com/lankuohsing/InternLM_notes/tree/main/lecture_1

**Comment by lyyilin** [comment]
欢迎大家参观我的博客：https://blog.csdn.net/qq_73668945/article/details/135490679

**Comment by liuwei16** [comment]
https://g5b4uupmby.feishu.cn/wiki/QFEKwUNrfi4bXukLAL9cDwEvnOc

**Comment by siruzhong** [comment]
1. 大语言模型简介：https://siru.notion.site/1-8460d6f348f04c718f8899076e4e57bd?pvs=4
2. 书生·浦语大模型全链路开源体系：https://siru.notion.site/2-afcbf81373974fb5a39a8134abfe0f96?pvs=4

**Comment by linkch0** [comment]
https://linkch0.github.io/courses/InternLM/lecture/#l1

**Comment by cyq2368** [comment]
https://mwsnr0a94gx.feishu.cn/docx/NUzAdHquIoApslxI1mOcLSiunQh?from=from_copylink

**Comment by leonfrank** [comment]
https://github.com/leonfrank/InternLM-HW/blob/main/lecture1/1.pdf

**Comment by vjsmmk** [comment]
https://github.com/vjsmmk/puyu/tree/main/1

**Comment by funcfang** [comment]
https://www.yuque.com/funcfang/ke5qbq/gsyi9bv7ln9nig2v?singleDoc# 《第一次课 - 大模型体系》

**Comment by miku8miku** [comment]
https://note.youdao.com/s/Tdifkhx8

**Comment by FlyFutureSky** [comment]
https://flyfuturesky.notion.site/Lecture1-c38a8e2a9e114f53a113c90ab5fddce4?pvs=4

**Comment by GradOpt** [comment]
https://github.com/OswaldHongyu/InternLM_notes/blob/main/lecture1.pdf

**Comment by mmb135** [comment]
https://www.yuque.com/r/note/ba031392-7a89-41c7-b596-fc6992a9af2e
